README.md

```diff
@@ -47,7 +47,7 @@ Examples and notebook tutorials are located at [Project-MONAI/tutorials](https:/
 Technical documentation is available at [docs.monai.io](https://docs.monai.io).
 
 ## Contributing
-For guidance on making a contribution to MONAI, see the [contributing guidelines](https://github.com/Project-MONAI/MONAI/blob/master/CONTRIBUTING.md).
+For guidance on making a contribution to MONAI, see the [contributing guidelines](https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md).
```

Comment:
The problem is this link will point to the `dev` version of `CONTRIBUTING.md` even in the `main` branch, maybe a relative URL should be used instead?